test/e2e/common/node/security_context.go

```diff
@@ -49,6 +49,43 @@ var _ = SIGDescribe("Security Context", func() {
 		podClient = f.PodClient()
 	})
 
```

Comment:
Can we add some more e2e tests?
1. Start more than one pod and check each gets a unique user namespace.
2. Test to check permissions are correct for the supported volume types.

Reply:
@mrunalp we can, but as I mentioned in the PR description it is kind of pointless to do it now. The CI doesn't run with container runtimes that support this, so it is a no-op to add them now.

We will definitely add them as soon as the patches are merged in container runtimes and included in a release: as soon as we can update the CI to use those versions, we will also add more tests here.

Does it make sense?

Reply:
> @mrunalp we can, but as I mentioned in the PR description it is kind of pointless to do it now. The CI doesn't run with container runtimes that support this, so it is a no-op to add them now.

Its important to provide a full suite of tests that can be used by those trying to enable this feature to confirm that it works.  These tests should be present in this PR and I'd like to hear about your results running them and an invocation example so someone else can reproduce the results.

Reply:
@deads2k thanks, didn't know. I called it out in the PR description but no one said that until ~now. I've added several tests in test/e2e/common/node/security_context.go.

Please let me know what you think :)